pages/dkp/kommander/1.4/release-notes/index.md

```diff
@@ -30,6 +30,11 @@ Rate limiting happens on a per-pull basis regardless of whether the pulled image
 
 For more information on addressing this limit, see [Docker hub rate limits](../operations/manage-docker-hub-rate-limits).
 
+## New features
+
+- The Projects tab can now show all of the current Helm Release charts, their chart version, and the names of the clusters.
+- Kommander now supports attaching existing, external clusters that have additional networking restrictions in place such as a DMZ, a proxy server, or being behind a firewall. More information is available at, [Attaching an Existing Kubernetes Cluster](https://docs.d2iq.com/dkp/kommander/1.4/clusters/attach-cluster/).
+
 ## Component versions
```

Comment:
We'll have to update these versions, too. I'm not sure where they all come from - but if you need me to check I can get them all.

Reply:
Yes please!